### ISSUE TITLE
Phosphor is not running on the window

### ISSUE THREAD

**Issue by raveenkm** [gmu-swe/phosphor#213]
I am able to build Phosphor on Windows 10.  But, I am unable to run the integration tests step.  First, it started complaining that JVM is not found. This is because hard coding "java" and "jlink" in the   InstrumentMojo.java class. This should be java.exe and jlink.exe on windows. After fixing this problem, I was facing other path related problem with Jlink. Especially directories in windows may contain spaces. This is causing a problem for jlink. Is it possible to make Phosphor complaint to windows 10 ? I am trying with Java 9.

**Comment by jon-bell** [comment]
I'm sorry - we do not have easy access to a windows 10 machine to test and debug this. However, I would welcome any pull requests that improve path handling to better support paths on windows with spaces.

**Comment by raveenkm** [comment]
Thanks. The path-related problem is due to Maven, which I think Maven has to fix (I guess). I have seen some StackOverflow issues reporting this problem (issues with maven handling of paths with Whitespaces).
Today, downloaded the Phosphor onto another window machine and installed it. (Compiled with java 11.0.22) and instrumented 1.8.0_281. The integration tests went through as described in the tool 's build page. Now I will carry the test I wanted to run.

However, I made the following changes to InstrumentUtil.java under the package edu.columbia.cs.psl.phosphor.agent package. (I am sorry I gave wrong class name in my previous post). This change is required to run the tests on Windows.

![image](https://github.com/gmu-swe/phosphor/assets/44434370/feac339c-ffe5-4446-80e4-01f0c016c902)
